From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Christopher A. Williamson <contact@chrisaw.io>
Date: Sun, 15 Feb 2026 00:00:00 +0000
Subject: [PATCH 5/5] Remove Auditor from build

Auditor relies on hardware attestation tied to specific device models
and will never pass verification on a GSI. Remove it from the product
packages to avoid shipping a non-functional app.

Change-Id: I0000000000000000000000000000000000000003
---
 target/product/handheld_product.mk | 1 -
 1 file changed, 1 deletion(-)

diff --git a/target/product/handheld_product.mk b/target/product/handheld_product.mk
--- a/target/product/handheld_product.mk
+++ b/target/product/handheld_product.mk
@@ -23,7 +23,6 @@
 # /product packages
 PRODUCT_PACKAGES += \
     AppStore \
-    Auditor \
     Calendar \
     Camera \
     Contacts \
-- 
2.34.1

